### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
From #2111 
@bacoords Everything is done 🍏

### CODE REVIEW

**Comment by szepeviktor** [comment]
Please consider using a package manager and a build process to get dependencies like Bootstrap.

File: inc/class-wp-bootstrap-navwalker.php [hunk 1]
```diff
@@ -407,7 +407,7 @@ public static function fallback( $args ) {
 		 *
 		 * @return array  $classes         a maybe modified array of classnames.
 		 */
-		private function seporate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
+		private function separate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
```
Comment (bacoords) [inline inc/class-wp-bootstrap-navwalker.php:410]: I think this is the only change I'd like to see removed from this PR. If we want to update the name of this function, I think we'd need to still keep the older one and deprecate it to be safe, but I think we'd want that in a separate PR.

**Review by bacoords — CHANGES_REQUESTED** [review]
Looks great! One change request, though open to feedback from others (CC @IanDelMar )

File: inc/class-wp-bootstrap-navwalker.php (hunk 1 above)
Comment (bacoords) [reply inc/class-wp-bootstrap-navwalker.php:410]: Just realizing we already had this discussion about this being a private function. So to be clear - we see no reason to add the old function with a deprecation notice, correct?

Comment (szepeviktor) [reply inc/class-wp-bootstrap-navwalker.php:410]: Yes, correct.

**Comment by szepeviktor** [comment]
@bacoords Included that new finding in this PR.

File: inc/class-wp-bootstrap-navwalker.php (hunk 1 above)
Comment (pattonwebz) [reply inc/class-wp-bootstrap-navwalker.php:410]: Noting here that once this is merged to understrap I would be more than happy to merge the same changeset to the navwalker repo as well. I have partially stepped away from that project, but I trust that the changes here are being tested and done for the best of reasons so would want to roll that back to the origin repo as well :)

**Review by bacoords — APPROVED** [review]

**Pull request merged**